example/aoBake.js

```diff
@@ -185,20 +66,19 @@ async function init() {
 	new OrbitControls( camera, renderer.domElement );
 	statusEl = document.getElementById( 'status' );
 
+	// const url = 'https://raw.githubusercontent.com/gkjohnson/3d-demo-data/main/models/stanford-bunny/bunny.glb';
```

Comment:
I noticed this bunny GLB seems to cause the UV generator step to get completely stuck. I'm not sure if it has something to do with the fact that this model has no UVs?

Reply:
That's strange, I haven't seen it lock up before. In another UV unwrapping task I had (unrelated to this project, not using xatlas) I did hit a case where adding non-zero padding setting caused things to get stuck because sometimes it's not tractable to introduce the requested amount of padding and also pack the texture. It's a long shot but maybe reducing the padding would help.

Reply:
Thanks - I'll have to try again and see if I can track down what was happening.